kaizen-openapi-parser/src/main/java/com/reprezen/kaizen/oasparser/types3.yaml

```diff
@@ -8,7 +8,7 @@ decls:
   - noextPat: &noextPat "(?!x-).*"
   - namePat: &namePat "[a-zA-Z0-9\\._-]+"
   - noextNamePat: &noextNamePat "(?!x-)[a-zA-Z0-9\\._-]+"
-  - pathPat: &pathPat "/.+"
+  - pathPat: &pathPat "/.*"
```

Comment:
@ghillairet , can you please also provide tests showing that the problem is now fixed?